### PR TITLE
Update FDZ_Data_Cleaning.Rmd

### PR DESCRIPTION
**Pull request by ruckeran**
added example for changeMissings() with multiple variables

### CODE REVIEW

File: vignettes/FDZ_Data_Cleaning.Rmd [hunk 1]
```diff
@@ -25,7 +25,7 @@ knitr::opts_chunk$set(echo = TRUE, eval = TRUE)
 The setup for this illustration is the following:
 ``` {r path}
 ## if necessary, install eatGADS and eatFDZ
-#install.packages(eatGADS)
+#install.packages("eatGADS")
```
Comment (beckerbenj) [inline vignettes/FDZ_Data_Cleaning.Rmd:28]: LGTM, thanks for spotting this!

File: vignettes/FDZ_Data_Cleaning.Rmd [hunk 2]
```diff
@@ -81,14 +81,21 @@ extractMeta(dat2, vars = c("school"))
...
-At times, only individual missings need to be tagged. `changeMissings()` tags missing values as missings by specifying them under `value` and marks them as `miss` under `missings.` This is also possible for multiple variables at the same time.
+At times, only individual missings need to be tagged. `changeMissings()` tags missing values as missings by specifying them under `value` and marks them as `miss` under `missings.`
 ```{r changeMissings}
 extractMeta(dat2, vars = c("info"))
 dat3 <- changeMissings(dat2, varName = "info", value = c("-97", "-98", "-99"), missings = c("miss", "miss", "miss"))
 extractMeta(dat3, vars = c("info"))
 ```
 The values `-97`, `-98` and `-99` are now tagged as `miss`.
 
+Changing the missing tags is also possible for multiple variables at the same time using `changeMissings()`.
```
Comment (beckerbenj) [inline vignettes/FDZ_Data_Cleaning.Rmd:92]: LGTM!

**Review by beckerbenj — APPROVED** [review]
Thanks for implementing this, everything LGTM!

**Comment by ruckeran** [comment]
Thank you for reviewing!

**Pull request merged**